connexion/operations/abstract.py

```diff
@@ -243,45 +228,20 @@ def get_mimetype(self):
             return DEFAULT_MIMETYPE
 
     @property
-    def _uri_parsing_decorator(self):
+    def uri_parser_class(self):
```

Comment:
Would prefer not to pass this via the operation. We can probably handle it similarly to the Jsonifier.